### ISSUE TITLE
[XCUITests] L10n Intro test needs to be updated

### ISSUE THREAD

**Issue by isabelrios** [mozilla-mobile/firefox-ios#11733]
The [test](https://github.com/mozilla-mobile/firefox-ios/blob/59568c9ea028c11959f8b02501c6503372385047/L10nSnapshotTests/L10nSuite1SnapshotTests.swift#L41) is failing because is waiting for the wallpapers onboarding slide. That one is not shown on Bitrise or locally. Is this the final design or is it an experiment? @yoanarios @thatswinnie asking you since I have seen some commits related to onboarding coming from you :) 

Current second and last onboarding card

![Simulator Screen Shot - iPhone 11 - 2022-08-29 at 13 45 28](https://user-images.githubusercontent.com/1897507/187196379-d2c6aff3-7843-48c8-b1b8-def0cbf9286f.png)

cc @jjSDET @clarmso

**Comment by yoanarios** [comment]
@isabelrios yes we are removing the wallpapers card from the onboarding plus other updates including design and text. I've updated `FirstRunTourTests` but I miss this one. This changes will be available in release v106 but already enabled in develop and beta

**Comment by isabelrios** [comment]
Thanks for your quick response @yoanarios !! that makes sense. That test belong to the l10n test suite which does not run regularly in PRs that's why you missed it, so no worries at all :) 

@jjSDET @clarmso That test will be fixed by doing something like:
```
func testIntro() {
        sleep(3)
        waitForExistence(app.scrollViews.staticTexts["WelcomeCardTitleLabel"], timeout: 15)
        snapshot("Onboarding-1")
        // Swipe to the second screen
        app.buttons["\(rootA11yId)PrimaryButton"].tap()
        currentScreen += 1
        waitForExistence(app.buttons["SignSyncCardSecondaryButton"])
        snapshot("Onboarding-2")
    }
```
